client/card-readers/settings/sections/contacts-details.js

```diff
@@ -26,12 +26,35 @@ const ContactDetailsSection = () => {
 		setAccountBusinessSupportPhone,
 	] = useAccountBusinessSupportPhone();
 
-	const businessSuppotEmailErrorMessage = useGetSavingError()?.data?.details
+	let businessSuppotEmailErrorMessage = useGetSavingError()?.data?.details
 		?.account_business_support_email?.message;
 
-	const businessSuppotPhoneErrorMessage = useGetSavingError()?.data?.details
+	let businessSuppotPhoneErrorMessage = useGetSavingError()?.data?.details
 		?.account_business_support_phone?.message;
 
+	if ( '' === accountBusinessSupportEmail ) {
+		businessSuppotEmailErrorMessage = __(
+			'Support email cannot be empty!',
+			'woocommerce-payments'
+		);
+	}
+
+	if ( '' === accountBusinessSupportPhone ) {
+		businessSuppotPhoneErrorMessage = __(
+			'Support phone number cannot be empty!',
+			'woocommerce-payments'
+		);
+	}
+
+	const updateSaveButtonAvailability = () => {
+		if (
```

Comment:
It seems that we are not accounting for the case when `accountBusinessSupportEmail` or `accountBusinessSupportPhone` are valid. I believe this prevents the 'Save changes' button to be enabled again.